lighthouse-core/audits/seo/font-size.js

```diff
@@ -5,7 +5,7 @@
  */
 'use strict';
 
-const parseURL = require('url').parse;
+const URL = require('../../lib/url-shim');
```

Comment:
I missed that `require('url').parse` won't work in the extension. Thankfully it was easy to replace.